spark/src/main/scala/org/apache/comet/expressions/CometCast.scala

```diff
@@ -136,7 +187,7 @@ object CometCast {
         // https://github.com/apache/datafusion-comet/issues/328
         Incompatible(Some("Not all valid formats are supported"))
       case _ =>
-        Unsupported
+        Unsupported(Some(s"Cast from String to $toType is not supported"))
```

Comment:
can see lots of Unsupported(Some(s"Cast from String to $toType is not supported")),  canthis  be a helper function? 

```
def unsupportedCast(from, to) {
   Unsupported(Some(s"Cast from $from to $to is not supported"))
}
```

Reply:
Updated. Thanks.